#
# Necessary to actually use BytesArray.
# Signed-Off: Michael 'Mickey' Lauer <mlauer@vanille-media.de>
#
Index: vala/vapi/glib-2.0.vapi
===================================================================
--- vala/vapi/glib-2.0.vapi	(Revision 2422)
+++ vala/vapi/glib-2.0.vapi	(Arbeitskopie)
@@ -3044,6 +3044,9 @@
 		public void sort (CompareFunc compare_func);
 		public void sort_with_data (CompareDataFunc compare_func);
 		public void set_size (uint length);
+
+		public uint len;
+		public uint8[] data;
 	}
 
 	/* N-ary Trees */
